### PR TITLE
Anime watch history is now saved to local history

### PR DESCRIPTION
**Pull request by zephex-alt**
And if you want to access this data, use the following code:

```js
	const [localItems, setLocalItems] = useState(null);

	useEffect(() => {
		const newData = get_local();
		setLocalItems(newData);
	}, []); // Empty dependency array means this effect runs only once after the initial render

	function get_local() {
		try {
			const data = localStorage.getItem("watchHistory");
			return JSON.parse(data);
		} catch (error) {
			console.log("error", error);
			return false;
		}
	}
```

### CODE REVIEW

**Comment by avalynndev** [comment]
You haven't created a LocalStorage File..?

```
Module not found: Can't resolve './LocalStorage'
```

**Comment by avalynndev** [comment]
Can you also use that Logic as a example in the page.tsx file, and create a card that represents the Name of the Anime And the the Episode Number?

**Comment by zephex-alt** [comment]
> You haven't created a LocalStorage File..?
> 
> ```
> Module not found: Can't resolve './LocalStorage'
> ```

Ohh, my bad.

I forgot to `git add *`

**Comment by zephex-alt** [comment]
_Check now_

**Review by avalynndev — APPROVED** [review]

File: components/EpisodeContainer.tsx [hunk 1]
```diff
@@ -2,107 +2,132 @@ import React, { useState } from "react";
 import Link from "next/link";
 import { SelectItem, Select, Button } from "@nextui-org/react"; // Assuming Button component is defined elsewhere
 import PetIcon from "@/components/Icons/PetIcon";
+import SaveToLocalStorage from "./LocalStorage";
```
[the remark this answers is not in the document]
Comment (zephex-alt) [reply components/EpisodeContainer.tsx:5]: > also can you define this LocalStorage file in utils folder (create one).

You can. Just make sure that you are importing it correctly.

**Pull request merged**